openpype/hosts/maya/api/lib.py

```diff
@@ -174,6 +174,22 @@ def maintained_selection():
             cmds.select(clear=True)
 
 
+def reload_textures(preset):
+    """Reload textures during playblast
+    """
+    if not preset["viewport_options"]["reloadTextures"]:
+        self.log.debug("Reload Textures during playblasting is disabled.")
+        return
```

Comment:
This taking `preset` as an argument is not very nice for the `lib` because it makes it hard to call this elsewhere.

```suggestion
def reload_textures():
    """Reload textures and regenerate UDIM tile previews"""
```